clang/test/CIR/CodeGen/variadics.c

```diff
@@ -3,8 +3,29 @@
 // RUN: %clang_cc1 -x c++ -std=c++20 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir %s -o %t.cir
 // RUN: FileCheck --input-file=%t.cir %s
 
-int average(int count, ...);
-// CHECK: cir.func private @{{.*}}average{{.*}}(!s32i, ...) -> !s32i
+#include <stdarg.h>
```

Comment:
Similar to other PR: get the mock version for the things you need from `stdarg.h`, should there be something to copy-n-paste from the tests!